scripts/configure_freebsd_ci_jail.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env sh
+
+set -o xtrace
```

Comment:
`set -e`?

Reply:
I planned to rewrite both of those shell scripts but forgot about them (didn't test upload either). Resetting to draft. Thanks!